src/endpoints/Stories/types.ts

```diff
@@ -180,6 +180,31 @@ export interface SlateStoryUpdateRequest extends GenericUpdateRequest {
     content_version?: number;
 }
 
+export interface Preview {
```

Comment:
`Response` suffix please.

```suggestion
export interface PreviewResponse {
```